packages/@aws-cdk/aws-config/test/rule.test.ts

```diff
@@ -417,4 +413,47 @@ describe('rule', () => {
     });
   });
 
-});
+  test('create two custom rule and one function', () => {
```

Comment:
[nit] `rule` --> `rules`

Reply:
thank you! Good job. The typo has been fixed.